src/test/java/pacman/GhostTest.java

```diff
@@ -50,56 +68,286 @@ public void testLive() {
     }
 
     @Test
-    public void testExistPacmanEatable() {
-        Ghost ghost = new Ghost(5, 5, true, walls);
-        Pacman pacman = new Pacman(5, 5, true, walls, dots);
-        boolean resultToCompare = ghost.existPacmanEatable(pacman);
-        assertEquals(true, resultToCompare);
+    public void testSetDirection() {
```

Comment:
Maybe here is posible to change the name to testSetDirectionOfGhost, especifying the test is for ghost